insights/parsers/ceilometer_log.py

```diff
@@ -32,7 +32,21 @@ class CeilometerCentralLog(LogFileOutput):
     Note:
         Please refer to its super-class ``LogFileOutput``
     """
-    pass
+    def get_after(self, timestamp, lines=None):
+        """
+        Get a list of lines after the given time stamp.
+
+        Parameters:
+            timestamp(datetime.datetime): log lines after this time are
+                returned.
+            lines(list): the list of log lines to search (e.g. from a get).
+                If not supplied, all available lines are searched.
+
+        Returns:
+            (list): The list of log lines with time stamps after the given
+                date and time.
```

Comment:
Subsequent lines for `Returns:` should be indented at the same level as the first line.